### PR TITLE
Sync Messages from Server

### PR DESCRIPTION
**Pull request by sachincool**
Fixes https://github.com/fossasia/susi_chromebot/issues/165

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `master` branch.
- [ ] The unit tests pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

- [ ] All the functions created/modified in this PR contain relevant docstrings.

#### Short description of what this resolves:
* Get The messages from the api
* Save to local storage and show
* Send messages to the api

#### Changes proposed in this pull request:

- Gets message from susi api
- Show messages from susi api
- Syncs messages to Susi api

### CODE REVIEW

**Comment by sachincool** [comment]
@ms10398  @stealthanthrax  @gabru-md  @mariobehling  please review. I have done the tests and fixed it accordingly.  the leftover changes are `time stamps` in the new messages.

File: src/js/login.js [hunk 1]
```diff
@@ -50,6 +42,71 @@ function showLoggedInBlock(show) {
...
+                var msgObj = {
+                    query: queryInside,
+                    answer: answerInside,
+                    queryTime: queryTimes,
+                    answerTime: answerTimes
+                }
+                messages.push(msgObj);
+                localStorage.setItem("messages", JSON.stringify(messages));
+                console.table(messages);
+
```
Comment (gabru-md) [inline src/js/login.js]: Please remove the extra spacing here.

File: src/js/login.js [hunk 2]
```diff
@@ -50,6 +42,71 @@ function showLoggedInBlock(show) {
...
+                var answerInside = queryAnswerPair;
+                var msgObj = {
+                    query: queryInside,
+                    answer: answerInside,
+                    queryTime: queryTimes,
+                    answerTime: answerTimes
+                }
+                messages.push(msgObj);
+                localStorage.setItem("messages", JSON.stringify(messages));
+                console.table(messages);
```
Comment (gabru-md) [inline src/js/login.js]: Why are you doing a `console.table` ?
Can you please elaborate the use it has here?

Comment (sachincool) [reply src/js/login.js]: i'll remove it.

**Comment by sachincool** [comment]
@gabru-md  done. please review again

**Pull request merged**